table/queries/derived_satellite_scans.sql

```diff
@@ -1,52 +1,52 @@
 # https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-6
 CREATE TEMP FUNCTION ClassifySatelliteRCode(rcode INTEGER) AS (
   CASE
-    WHEN rcode = 1 THEN "⚠️dns/rcode:FormErr"
-    WHEN rcode = 2 THEN "⚠️dns/rcode:ServFail"
+    WHEN rcode = 1 THEN "❗️dns/rcode:FormErr"
+    WHEN rcode = 2 THEN "❓dns/rcode:ServFail"
     WHEN rcode = 3 THEN "❗️dns/rcode:NXDomain"
-    WHEN rcode = 4 THEN "⚠️dns/rcode:NotImp"
+    WHEN rcode = 4 THEN "❗️dns/rcode:NotImp"
     WHEN rcode = 5 THEN "❗️dns/rcode:Refused"
-    WHEN rcode = 6 THEN "⚠️dns/rcode:YXDomain"
-    WHEN rcode = 7 THEN "⚠️dns/rcode:YXRRSet"
-    WHEN rcode = 8 THEN "⚠️dns/rcode:NXRRSet"
-    WHEN rcode = 9 THEN "⚠️dns/rcode:NotAuth"
-    WHEN rcode = 10 THEN "⚠️dns/rcode:NotZone"
-    WHEN rcode = 11 THEN "⚠️dns/rcode:DSOTYPENI"
-    WHEN rcode = 12 THEN "⚠️dns/rcode:Unassigned"
-    WHEN rcode = 13 THEN "⚠️dns/rcode:Unassigned"
-    WHEN rcode = 14 THEN "⚠️dns/rcode:Unassigned"
-    WHEN rcode = 15 THEN "⚠️dns/rcode:Unassigned"
-    WHEN rcode = 16 THEN "⚠️dns/rcode:BadVers"
-    WHEN rcode = 17 THEN "⚠️dns/rcode:BadSig"
-    WHEN rcode = 18 THEN "⚠️dns/rcode:BadKey"
-    WHEN rcode = 19 THEN "⚠️dns/rcode:BadTime"
-    WHEN rcode = 20 THEN "⚠️dns/rcode:BadMode"
-    WHEN rcode = 21 THEN "⚠️dns/rcode:BadAlg"
-    WHEN rcode = 22 THEN "⚠️dns/rcode:BadTrunc"
-    WHEN rcode = 23 THEN "⚠️dns/rcode:BadCookie"
-    ELSE CONCAT("⚠️dns/unknown_rcode:", rcode)
+    WHEN rcode = 6 THEN "❗️dns/rcode:YXDomain"
+    WHEN rcode = 7 THEN "❗️dns/rcode:YXRRSet"
+    WHEN rcode = 8 THEN "❗️dns/rcode:NXRRSet"
+    WHEN rcode = 9 THEN "❗️dns/rcode:NotAuth"
+    WHEN rcode = 10 THEN "❗️dns/rcode:NotZone"
+    WHEN rcode = 11 THEN "❗️dns/rcode:DSOTYPENI"
+    WHEN rcode = 12 THEN "❗️dns/rcode:Unassigned"
+    WHEN rcode = 13 THEN "❗️dns/rcode:Unassigned"
+    WHEN rcode = 14 THEN "❗️dns/rcode:Unassigned"
+    WHEN rcode = 15 THEN "❗️dns/rcode:Unassigned"
```

Comment:
Are we sure that these are all cases of censorship?

Reply:
We are not, but we also don't have evidence that this is problematic. So it takes the ❗️ marker as per the criteria I mentioned.